src/init/common.cpp

```diff
@@ -77,6 +78,7 @@ void AddLoggingArgs(ArgsManager& argsman)
     argsman.AddArg("-logtimemicros", strprintf("Add microsecond precision to debug timestamps (default: %u)", DEFAULT_LOGTIMEMICROS), ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::DEBUG_TEST);
     argsman.AddArg("-printtoconsole", "Send trace/debug info to console (default: 1 when no -daemon. To disable logging to file, set -nodebuglogfile)", ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
     argsman.AddArg("-shrinkdebugfile", "Shrink debug.log file on client startup (default: 1 when no -debug)", ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
+    argsman.AddArg("-loglevel=<level>|<category>:<level>", strprintf("Sets the threshold log level and any logs with severity lower than the threshold level will be ignored (default -loglevel=info). -loglevel=<level> sets the global threshold log level and will be applied to all categories. If <category>:<level> is supplied, the threshold level overrides the global threshold level and will only be applied to <category>. <category> can be: %s, <level> can be: debug, info, warning, error. This option can be specified multiple times to set multiple category specific threshold level.", LogInstance().LogCategoriesString()), ArgsManager::DISALLOW_NEGATION | ArgsManager::DISALLOW_ELISION, OptionsCategory::DEBUG_TEST);
```

Comment:
```suggestion
    argsman.AddArg("-loglevel=<level>|<category>:<level>", strprintf("Set the global or per-category logging severity level; log messages of lower severity will not be printed to the debug log (default=info). If <category>:<level> is supplied, the setting will override the global one. <category> can be: %s, <level> can be: debug, info, warning, error. This option may be specified multiple times to set multiple category-specific levels.", LogInstance().LogCategoriesString()), ArgsManager::DISALLOW_NEGATION | ArgsManager::DISALLOW_ELISION, OptionsCategory::DEBUG_TEST);
```
- also suggest moving this line to just after `-logips` and writing a method like `LogCategoriesString()` to output the log levels `debug, none, info, warning, error` here, in SetLoggingLevel(), and in the future update to the logging RPC.

Reply:
Done 👍🏻. Moved the line to after `-logips`, used `constexpr` for default level and added a `LogCategoriesString`